Orange/evaluation/scoring.py

```diff
@@ -349,7 +349,7 @@ def graph_ranks(avranks, names, cd=None, cdmethod=None, lowv=None, highv=None,
     """
     Draws a CD graph, which is used to display  the differences in methods'
     performance. See Janez Demsar, Statistical Comparisons of Classifiers over
-    Multiple Data Sets, 7(Jan):1--30, 2006.
+    Multiple Datasets, 7(Jan):1--30, 2006.
```

Comment:
Please inform the Journal of Machine Learning Research that we decided to change the title of the paper.